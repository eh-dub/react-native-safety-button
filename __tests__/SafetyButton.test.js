import React from "react";
import renderer from "react-test-renderer";

import SafetyButton from "../";

test("renders correctly", () => {
  const tree = renderer.create(<SafetyButton />).toJSON();
  expect(tree).toMatchSnapshot();
});
