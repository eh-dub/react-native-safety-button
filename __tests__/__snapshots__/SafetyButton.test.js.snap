// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  accessible={true}
  focusable={false}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
>
  <View
    style={
      Object {
        "backgroundColor": "white",
        "bottom": 0,
        "left": 0,
        "position": "absolute",
        "right": 0,
        "top": 0,
        "width": "0%",
      }
    }
  />
</View>
`;
